v-next/hardhat/test/helpers/env-vars.ts

```diff
@@ -0,0 +1,34 @@
+import { afterEach } from "node:test";
+
+export function createTestEnvManager() {
```

Comment:
I've copied this from core. I think we need to move all these to a test utils module. We already discussed this with Pato, but I wanted to leave a note for awareness.

Reply:
Removed as it is no longer used.